docs/installing-solidity.rst

```diff
@@ -131,6 +131,10 @@ with the most recent changes, please use the following:
 
     sudo snap install solc --edge
 
+Note that the ``solc`` snap uses strict confinement. This is the most secure mode
```

Comment:
Can you change this into `.. note::`  so it is nicely formatted?

Reply:
Sure. Done.